force-app/main/default/classes/AFFL_MultiRecordType_TDTM.cls

```diff
@@ -267,17 +410,32 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
         for (Affiliation__c otherAffl : oldPrimaryAffls) {
             otherAffl.Primary__c = false;
         }
-        dmlWrapper.objectsToUpdate.addAll((List<SObject>)oldPrimaryAffls);
+        dmlWrapper.objectsToUpdate.addAll((List<SObject>) oldPrimaryAffls);
+        return oldPrimaryAffls;
     }
 
-    private void createProgramEnrollmentIfNecessary(Affiliation__c affl, DmlWrapper dmlWrapper) {
+    /*******************************************************************************************************
```

Comment:
Changes on this method include adding apex doc comments and having the method return a value for testability.